spec/requests/users_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /users' do
+    it 'should responce with success.' do
+      get users_path
+      expect(response).to have_http_status(:success)
+    end
+    it 'should responce with correct status' do
+      get users_path
+      expect(response.status).to eq(200)
+    end
+    it 'should include correct placeholder text.' do
+      get users_path
+      expect(response.body).to include('Users List')
+    end
+    it 'should render correct template' do
+      get users_path
+      expect(response).to render_template('index')
+    end
+  end
```

Comment:
- [ ] Good job on implementing all `user#index` action tests. Kindly also add all tests for the `user#show` action as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=Make%20sure%20that%20for%20each%20action%20you%20check%3A). All action should be tested.